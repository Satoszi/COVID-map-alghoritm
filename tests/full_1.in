6 6 2 8
full_1
++++++
++++++
++++++
++++++
++++++
++++++
