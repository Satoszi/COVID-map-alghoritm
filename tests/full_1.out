3 5
0 5
5 4
1 3
4 2
2 1
5 0
0 0
